dom.bs

```diff
@@ -1225,12 +1240,28 @@ for discussion).
    <li><p>Otherwise, set <var>target</var> to <var>parent</var> and append (<var>parent</var>,
    <var>target</var>, <var>relatedTarget</var>) to <var>event</var>'s <a for=Event>path</a>.
 
-   <li><p>If <var>parent</var> is non-null, then set <var>parent</var> to the result of invoking
-   <var>parent</var>'s <a>get the parent</a> with <var>event</var>.
+   <li>
+    <p>If <var>parent</var> is non-null, then:
+
+    <ol>
+     <li><p>If <var>event</var>'s {{Event/bubbles}} attribute is true, <var>activationTarget</var>
+     is null, and <var>parent</var> has <a>activation behavior</a>, then set
+     <var>activationTarget</var> to <var>parent</var>.
+     <!-- If an EventTarget object with activation behavior can ever have an associated shadow root
+          we need to complicate this somewhat. -->
```

Comment:
The alternative is changing both substeps that append to event's path to have this kind of check. But if we never add activation behavior to shadow hosts, that would be needlessly complicated. But maybe I should do it anyway just to be forward compatible.